client.go

```diff
@@ -3,6 +3,7 @@ package singularity
 import (
 	"net/http"
 
+	"github.com/opentable/sous/util/logging"
```

Comment:
If this import is just for the `logging.LogSink` interface, might be better to simply copy the interface here to avoid explicit coupling.